### DISCUSSION TITLE
Changing staff skins

### DISCUSSION THREAD

**Discussion by Liampie** [OpenRCT2/OpenRCT2#15690] [Feature requests] [answered]
Using the most recent 8cars trainer, we could change any staff member into any other type of staff members.

Most frequent use of this was to turn pandas, tigers and elephants into cowboys, pirates, gorillas and other entertainers that weren't available to the bench because the scenery tabs weren't there. I think this is already covered by OpenRCT.

However, there is more. We could also change entertainers into mechanics, handymen into security, mechanics into handymen, and whatever you could think of. What's cool about this is that you could get handymen or mechanics in different colours this way. If your handymen are red, and your mechanics are blue, you can get a red mechanic by changing one of the handymen's skin into a mechanic.

Since they're almost always used as frozen scenery pieces I'm not sure how they function, but for the looks alone this would be nice to have.

Even better would it be if we could pick the uniform colour for each staff member individually, using a new paint tab in a staff member's window, as if it were a ride. Would be awesome and I can see many people doing great stuff with that.

**Comment by Ryder17z** [comment]
Would be really nice to have pandas as handymen and Tigers as mechanics and such.

**Comment by Gymnasiast** [comment]
I don't think the last option is currently possible. It could be made an option when we should to our own format, provided there is demand for it. Considering the other things were possible with 8 cars, they can be implemented in OpenRCT2 as it is.

**Comment by janisozaur** [comment]
Don't we already have that? There is a console command to change the entertainer costume, does it solve the stated problem?

**Comment by duncanspumpkin** [comment]
The console command is only for entertainers i think. Wouldn't be hard to enhance it to allow the above changes.

**Comment by AaronVanGeffen** [comment, marked as answer]
This has been possible with @Manticore-007's Peep Editor for a while.
